sentences/de/_common.yaml

```
@@ -183,7 +183,7 @@ expansion_rules:
   luefter: "<artikel> (Ventilator[en]|Lüfter)"
   alle_luefter: "<alle> (Ventilatoren|Lüfter)"
   brightness: "{brightness} [Prozent|%]"
-  temperature: "{temperature} [Grad] [{temperature_unit}]"
+  temperature: "{temperature} [Grad] [[°]{temperature_unit}]"
```

```suggestion
  temperature: "{temperature}[ ][°|Grad][ ][{temperature_unit}]"
```

i think we should adopt the English approach
https://github.com/home-assistant/intents/blob/bfe1eb8feb4a6ed623c4fcdd670fc200f52f23a2/sentences/en/_common.yaml#L304

---

I agree :+1: